packages/webpack-model/src/index.ts

```diff
@@ -36,7 +36,7 @@ export function prepareWithJora(
   return {
     files,
     compilations: compilations.map((c) => c.data),
-    query: (query: string, data: unknown, context: unknown = {}): unknown =>
+    query: (query: string, data: unknown, context: unknown = rootContext): unknown =>
```

Comment:
You're forgot `const rootContext = {};` before `return` ;)

Reply:
Sorry! Thank you for your patience. And for the great tool too :)